File: 0009-palindrome-number/0009-palindrome-number.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
    if (x < 0) {
        return false;
    }
    let temp=0;
    let num=x;
    while(x!=0){
        let rem=x%10;
        temp=temp*10+rem;
        x=Math.floor(x/10);
    }
    if(num==temp){
        return true;
    }else{
        return false;
    }
};

File: 0009-palindrome-number/0009-palindrome-number.java
class Solution
{
    public boolean isPalindrome(int x)
    { 
        if(x<0){
            return false;
        }
        int rev=0,rem;
        int temp=x;
        while(temp>0)
        {
            rem=temp%10;
            rev=rev*10+rem;
            temp=temp/10;
            
        }
        return rev==x;
    }
}